src/ChannelServer/Network/ChannelPacketSender.cs

```diff
@@ -1275,11 +1275,12 @@ public static void ZC_EXP_UP_BY_MONSTER(Character character, int exp, int jobExp
 		/// </summary>
 		/// <param name="character"></param>
 		/// <param name="exp"></param>
-		public static void ZC_EXP_UP(Character character, int exp)
+		/// <param name="jobExperience"></param>
+		public static void ZC_EXP_UP(Character character, int exp, int jobExperience)
```

Comment:
Just a pointer, but this would've made a nice separate commit, now it's kinda hidden in between this:

![ss 2016-04-06 at 09 55 03](https://cloud.githubusercontent.com/assets/5593920/14330783/73412f14-fc42-11e5-8a27-ce515932ad13.png)